src/environment.cpp

```diff
@@ -1064,7 +1064,8 @@ void ServerEnvironment::step(float dtime)
 	/*
 		Manage active block list
 	*/
-	if(m_active_blocks_management_interval.step(dtime, 2.0))
+	float abm_interval = g_settings->getFloat("abm_interval");
```

Comment:
@nerzhul I don't see why this would be dangerous, modification would come through a chatcommand, and not apply until the next ```::step```, so nothing unsafe about that.

The same function already uses settings->get several times on these:
- dedicated_server_step
- active_block_range
and those aren't cached either.

Given @celeron55's comment, you'll have to make up your mind together, I couldn't really care either way (uncached is fine, but then there should be no uncached settings there either), but I can't change the code BOTH ways.